src/sentry/static/sentry/app/views/organizationDiscover/discover.jsx

```diff
@@ -26,15 +38,21 @@ import {
   TopBar,
   Sidebar,
   SidebarTabs,
+  SavedQueryTitle,
+  SavedQueryAction,
   PageTitle,
+  EditableName,
 } from './styles';
 
 import {trackQuery} from './analytics';
 
 export default class OrganizationDiscover extends React.Component {
   static propTypes = {
-    organization: SentryTypes.Organization,
-    queryBuilder: PropTypes.object,
+    organization: SentryTypes.Organization.isRequired,
+    queryBuilder: PropTypes.object.isRequired,
+    savedQuery: PropTypes.object, // Provided if it's a saved search
```

Comment:
Can we change this to `shape` and add this type to `sentryTypes`?

Reply:
done